roles/vault/tasks/main.yml

```diff
@@ -43,3 +43,32 @@
 # Post install
 - name: Post install
   ansible.builtin.include_tasks: post-install.yml
+
+- name: Add helm repo
+  kubernetes.core.helm_repository:
+    name: dso
+    repo_url: "{{ dsc.global.backup.vault.helmRepoUrl }}"
+    force_update: true
+  when: dsc.global.backup.vault.enabled
+
+- name: Deploy helm
+  kubernetes.core.helm:
+    name: "{{ dsc_name }}-vault-backup"
+    chart_ref: dso/cpn-backup-utils
+    chart_version: "{{ dsc.global.backup.vault.chartVersion }}"
+    release_namespace: "{{ dsc.vault.namespace }}"
+    values:
+      vault:
+        enabled: true
+        secrets:
+          S3_BUCKET_NAME: "{{ dsc.global.backup.s3.bucketName }}"
+          S3_BUCKET_PREFIX: "{{ dsc.global.backup.vault.pathPrefix }}"
+          S3_ENDPOINT: "{{ dsc.global.backup.s3.endpointURL }}"
+          VAULT_ADDR: "{{ vault_domain }}"
+          VAULT_TOKEN: "{{ vault_token }}"
+          S3_ACCESS_KEY: "{{ dsc.global.backup.s3.credentials.accessKeyId.value | b64encode }}"
+          S3_SECRET_KEY: "{{ dsc.global.backup.s3.credentials.secretAccessKey.value | b64encode }}"
+          RETENTION: "{{ dsc.global.backup.vault.retentionPolicy | b64encode }}"
+        job:
+          schedule: "{{ dsc.global.backup.vault.cron }}"
+    state: "{{ dsc.global.backup.vault.enabled | ternary('present', 'absent') }}"
```

Comment:
Use of FQCN ? 
```suggestion
    state: "{{ dsc.global.backup.vault.enabled | ansible.builtin.ternary('present', 'absent') }}"
```

Reply:
Bon bah y'en a déjà d'autres qui n'ont pas le `ansible.builtin`, je m'aligne pour l'instant on fera un coup de clean sur le dépôt un de ces quatre